GOVERNANCE.md

```diff
@@ -0,0 +1,71 @@
+# Appium Project Governance
+
+The Appium Project wants as much as possible to operate using procedures that
+are fair, open, inviting, and ultimately good for the community. For that
+reason we find it valuable to codify some of the ways that the Project goes
+about its day-to-day business. We want to make sure that no matter who you are,
+you have the opportunity to contribute to Appium. We want to make sure that no
+one corporation can exert undue influence on the community or hold the Project
+hostage. And likewise we want to make sure that corporations which benefit from
+Appium are also incentivized to give back.
+
+### The Technical Committee
+
+The project is officially led by a Technical Committee (TC), currently
+consisting of [@jlipps](https://github.com/jlipps),
+[@bootstraponline](https://github.com/bootstraponline),
+[@penguinho](https://github.com/penguinho), and
+[@dandoveralba](https://github.com/dandoveralba), who all represent different
+companies and different types of contribution to the Appium Project. The TC is
+responsible for high-level decisions (like establishing a feature roadmap,
+accepting sponsorship, organizing conferences, etc...), adjudicating conflicts
+between other members of the community, and making changes to the project's
```

Comment:
"other" here reads oddly to me. Like there will never be conflicts between TC members.